ui/src/message_popup/author/new_question_page.jsx

```diff
@@ -1,3 +1,4 @@
+/* eslint no-console: "off" */
```

Comment:
Didn't exactly remove flow here but just didn't add it for the same reason I noted above.